var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -17,7 +17,7 @@ class Petsc(Package):
 
     homepage = "http://www.mcs.anl.gov/petsc/index.html"
     url      = "http://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-3.5.3.tar.gz"
-    git      = "https://bitbucket.org/petsc/petsc.git"
+    git      = "https://gitlab.com/petsc/petsc.git"
 
     maintainers = ['balay', 'barrysmith', 'jedbrown']
 
```

Comment:
You can change the `develop` version to `master` now that #1983 has been merged.

Reply:
Hm - that would  require updates to other packages aswell [wrt dependencies].
And I don't see `maint` in that list [wrt petsc, maint is the release branch - and master is the development branch]

I guess I should make the switch at some point.